frontend/src/app/modules/settings/team/team-preferences/team-preferences.component.ts

```diff
@@ -134,6 +150,15 @@ export class TeamPreferencesComponent extends BaseComponent implements OnInit {
         control.patchValue(removeExcessiveSpaces(control.value));
     }
 
+    public deleteLogo() {
+        if (this.team?.id) {
+            this.teamService.deleteLogo(this.team.id)
+                .pipe(this.untilThis)
+                .subscribe();
+        }
+        this.imageUrl = '';
```

Comment:
why not in subscribe?

Reply:
We need to set it as empty during team creation as well

Reply:
how the deleteLogo method related to the team creation process?